src/content/docs/cypher/data-definition/create-table.md

```diff
@@ -83,24 +91,45 @@ Output:
 | City            | NODE      |              |
 ----------------------------------------------
 ```
+
 A relationship table group can be used as a regular relationship table for querying purposes.
+
 ```cypher
 MATCH (a:User)-[:Knows]->(b) RETURN *;
 ```
+
 The query above is equivalent to the following:
+
 ```cypher
 MATCH (a:User)-[:Knows_User_User|:Knows_User_city]->(b) RETURN *;
 ```
 
 As you can imagine, the more relationships you want to selectively query on, the more useful relationship table groups become.
 
+## Create sequence
```

Comment:
I'm not sure if this is the right place for sequence, should it be mixed with create table statements?

Reply:
@prrao87 should probably decide since he probably wants to change/add to what I wrote anyways.

Reply:
Sure, will take a look and decide where to place it.